accel-pppd/ctrl/pppoe/pppoe.c

```diff
@@ -1458,6 +1534,14 @@ static void __pppoe_server_start(const char *ifname, const char *opt, void *cli,
 		return;
 	}
 
+#ifndef HAVE_VPP
+	if (is_non_dev_ppp) {
+		log_warn("pppoe: No support for vpp, option vpp-cp ignored for interface %s\r\n", ifname);
```

Comment:
I'd argue that if people chose to build accel-ppp without VPP support, VPP-specific options should be treated as parse errors if possible. If accel-ppp's general approach to unknown options is to ignore them, then we should ignore them to be consistent with that.